simple-service-webapp-tests/src/test/java/com/example/TestTest.java

```diff
@@ -112,58 +112,33 @@ public void testPutMethod() throws IOException,ParseException {
     }
 
     @Test
-    public void getLoginToken() throws IOException,ParseException {
```

Comment:
This is not actually a test, it's just a utility to set the token for other test methods. So it would be cool if there was a method with a JUnit BeforeClass annotation that sets the token field, then all other test could